RecoJets/JetProducers/interface/PileupJetIdAlgo.h

```diff
@@ -37,6 +37,8 @@ class PileupJetIdAlgo {
 					       float jec, const reco::Vertex *, const reco::VertexCollection &, double rho);
 
 	void set(const PileupJetIdentifier &);
+        std::unique_ptr<const GBRForest> getMVA(std::vector<std::string>, const std::string &);
```

Comment:
The first parameter should actually be a const reference (I forgot to say so in my original coment) to avoid copying overhead:
```
std::unique_ptr<const GBRForest> getMVA(const std::vector<std::string> &, const std::string &);
```
Same for getMVAvars.